### PR TITLE
Test pull request (PR)

### PR DESCRIPTION
**Pull request by mgrinko**
Just to show how it works

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
```
Comment (mgrinko) [inline index.html]: Hello

Comment (samvimes01) [reply index.html]: Hi

Comment (Ezio17) [reply index.html]: Hi

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  <title>Title</title>
+</head>
```
Comment (Valeriy-Sorochynskyi) [inline index.html]: Hello

File: index.html (hunk 1 above)
Comment (nastasia17) [inline index.html]: test

File: index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (17Octopus17) [inline index.html]: good job

File: index.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+<head>
```
Comment (OlgaTitarenko) [inline index.html]: comm

File: index.html (hunk 3 above)
Comment (UlyanaShnyr) [inline index.html]: hi

File: index.html (hunk 1 above)
Comment (Valeria38) [reply index.html]: :)

File: index.html (hunk 3 above)
Comment (zakharovvu) [inline index.html]: Добавим 'ru

File: index.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  <meta charset="UTF-8">
+  <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (Andreas-Just) [inline index.html]: finish

File: index.html [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Title</title>
+</head>
+<body>
```
Comment (olegobiukh) [inline index.html]: I was here

File: index.html (hunk 3 above)
Comment (Maryana92) [inline index.html]: Ні!

File: index.html [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (bogdanpashynskyi) [inline index.html]: this body is sad because it doesn't have any elements :(

File: index.html (hunk 2 above)
Comment (Elizasuschenko) [inline index.html]: ///

File: index.html [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (OzirnaRita) [inline index.html]: Привет

File: index.html (hunk 3 above)
Comment (Anet199516) [reply index.html]: Helllo!

File: index.html (hunk 4 above)
Comment (BohWebo) [inline index.html]: hi

**Pull request merged**